main.tf

```diff
@@ -25,8 +25,8 @@ data "google_project" "project" {
 }
 
 locals {
+  default_subscription_label   = { "managed_by" : "terraform" }
   default_ack_deadline_seconds = 10
```

Comment:
Doesn't look like these locals are being used. Though I would love us to append the subscription and topic labels with the managed_by label instead of having to ensure a user puts it in the config. We would also probably like to have defaults for ack deadline but I know the data structure is kind of weird.

Reply:
Maybe we can use https://www.terraform.io/language/functions/merge

Reply:
local.default_ack_deadline_seconds is used when not added to the .tf file. I just added it to the .tf files in the other pr as part of my generation script. And I can look into the merge function.

Reply:
Was able to move the merge. But because of us not wanting to use the expiramental option type. You still need to include the `subscription_label = {}`